packager/packager.js

```diff
@@ -15,24 +15,26 @@ var http = require('http');
 
 var getFlowTypeCheckMiddleware = require('./getFlowTypeCheckMiddleware');
 
-if (!fs.existsSync(path.resolve(__dirname, '..', 'node_modules'))) {
+try {
+  var chalk = require('chalk');
+  var connect = require('connect');
+  var ReactPackager = require('./react-packager');
+  var blacklist = require('./blacklist.js');
+  var launchEditor = require('./launchEditor.js');
+  var parseCommandLine = require('./parseCommandLine.js');
+  var webSocketProxy = require('./webSocketProxy.js');
+} catch (err) {
```

Comment:
Can you check if the error has the string `Cannot find module ` before going on with the following code. Otherwise we should rethrow the error to make sure we don't miss important things.